modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -60,9 +60,9 @@ $ ./openshift-install version
 .Example output for a shared registry binary
 [source,terminal,subs="quotes"]
 ----
-./openshift-install 4.18.0
+./openshift-install 4.19.0
 built from commit ae7977b7d1ca908674a0d45c5c243c766fa4b2ca
-release image registry.ci.openshift.org/origin/release:4.18ocp-release@sha256:0da6316466d60a3a4535d5fed3589feb0391989982fba59d47d4c729912d6363
+release image registry.ci.openshift.org/origin/release:4.19ocp-release@sha256:0da6316466d60a3a4535d5fed3589feb0391989982fba59d47d4c729912d6363
```

Comment:
also here. In this case, swapping only the version numbers probably makes the rest of this output technically an incorrect example, but I'm not sure anyone is going to check whether the example output is _exactly_ accurate 🤷‍♂️
What matters most is that the output illustrates an example of the format they could see in response to the command

```suggestion
release image registry.ci.openshift.org/origin/release:{product-version}ocp-release@sha256:0da6316466d60a3a4535d5fed3589feb0391989982fba59d47d4c729912d6363
```